cli/query/query.go

```diff
@@ -21,36 +23,76 @@ func QueryProof() {
 	req := make(map[string]string)
 	req["platform"] = params.Platform
 	req["identity"] = params.Identity
-	req["page"] = cast.ToString(params.Page)
+	req["page"] = params.Page
+	getAndPrintData(req)
 
+	input := bufio.NewScanner(os.Stdin)
+	for {
+		fmt.Println("\nChoose next step:\n 1. next page\n 2. Get the data of the specified page\n 3. Quit\n Enter the number:")
+		input.Scan()
+		nextStep := input.Text()
+		switch nextStep {
+		case "1":
+			page, _ := strconv.Atoi(params.Page)
+			page += 1
+			fmt.Printf("\nGet the data of Page %d\n", page)
+			req["page"] = strconv.FormatInt(int64(page), 10)
+			getAndPrintData(req)
+		case "2":
+			fmt.Println("\nPlease enter the page number")
+			input.Scan()
+			page := input.Text()
+			req["page"] = page
+			getAndPrintData(req)
+		case "3":
+			os.Exit(1)
```

Comment:
This `os.Exit()` is requested by user, so it should not be considered as a `quit with error` .
`os.Exit(0)` instead.

Reply:
fixed.